includes/polling/os/hpvc.inc.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+// Format of sysDescr is hardware model followed by version followed by build date
+$tempstr = substr($poll_device['sysDescr'], 0, strrpos($poll_device['sysDescr'], ' '));
+$version = trim(substr($tempstr, strrpos($tempstr, ' ')));
+$hardware = trim(substr($tempstr, 0, strrpos($tempstr, ' ')));
+
+// Serial number is in sysName after string "VCEX"
+$serial = substr(snmp_get($device, 'sysName.0', '-OvQ', 'SNMPv2-MIB:HOST-RESOURCES-MIB:SNMP-FRAMEWORK-MIB'), 4);
```

Comment:
Yes, though looking at the code it gets transformed to lowercase from the original string. I wanted to keep the serial number in exactly the same format it's visible in other places. However, I can change this, if needed.